projects/common/src/time/time.ts

```diff
@@ -32,6 +32,19 @@ export class Time {
     return this._date;
   }
 
+  public static parse(time: string, isUTC: boolean = false): Time {
```

Comment:
The time string will have the time zone encoded in it - shouldn't be a separate param. For example in your test above, you gave 4 utc times, and only had it treat 2 of them as utc - the other two give the appear to be local times, but that's incorrect.

I would suggest you produce a UTC `Time` object out of the time string regardless of it's timezone:
```ts
    return new Time(
      scheduledDate.getUTCHours(),
      scheduledDate.getUTCMinutes(),
      scheduledDate.getUTCSeconds(),
      scheduledDate.getUTCMilliseconds(),
      true
    );
```

Reply:
oh okay

Reply:
updated